src/ck-ldb/BaseLB.h

```diff
@@ -150,18 +149,21 @@ class BaseLB: public CBase_BaseLB
 
   BaseLB(const CkLBOptions &opt)  { initLB(opt); }
   BaseLB(CkMigrateMessage *m):CBase_BaseLB(m) {
-    theLbdb = CProxy_LBDatabase(_lbdb).ckLocalBranch();
+    lbmgr = CProxy_LBManager(_lbmgr).ckLocalBranch();
   }
   virtual ~BaseLB();
 
   void unregister(); 
   inline const char *lbName() { return lbname; }
-  inline int step() { return theLbdb->step(); }
+  inline int step() { return lbmgr->step(); }
   virtual void turnOff() { CmiAbort("turnOff not implemented"); }
```

Comment:
For classes that inherit from BaseLB we should use C++11 "override" on their virtual functions and potentially "final" on the classes themselves (if users aren't allowed to inherit from them).

Reply:
Agreed, but I think this is something for a later change.